content/22.vaccines.md

```diff
@@ -365,6 +365,21 @@ The main advantage of this method is that they are considered very safe because
 
 #### Novavax NVX-CoV2373
 
+Novavax-CoV2373 is a protein nanoparticle vaccine candidate against SARS-CoV-2.
+The vaccine is constructed from a mutated SARS-CoV-2 spike protein in combination with a specialized adjuvant to elicit an immune response against SARS-CoV-2.
+The spike protein is recombinantly expressed in Sf9 insect cells[@doi: 10.1038/s41467-020-20653-8], which have previously been used for several other FDA-approved protein therapeutics [@doi:10.1021/acs.iecr.8b00985].
+The expressed spike protein contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) to avoid cleavage of the spike protein as well as two proline substitutions (K986P and V987P) to improve thermostability [@doi: 10.1038/s41467-020-20653-8].
```

Comment:
```suggestion
The expressed spike protein contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) to avoid cleavage of the spike protein as well as two proline substitutions (K986P and V987P) to improve thermostability [@doi:10.1038/s41467-020-20653-8].
```